packages/elements/src/react/common/form/index.tsx

```diff
@@ -161,12 +178,10 @@ const useField = ({ name }: Partial<Pick<FieldDetails, 'name'>>) => {
 
   const shouldBeHidden = false; // TODO: Implement clerk-js utils
   const hasError = feedback ? feedback.type === 'error' : false;
-  const validity = hasError ? FIELD_VALIDITY.invalid : FIELD_VALIDITY.valid;
 
   return {
     hasValue,
     props: {
-      [`data-${validity}`]: true,
```

Comment:
Radix already adds `data-valid`/`data-invalid` correctly to the field, so this is removed to not end up in a situation where both can exist at the same time.